Index: src/prova.java
===================================================================
diff --git a/src/prova.java b/src/prova.java
deleted file mode 100644
--- a/src/prova.java	
+++ /dev/null	
@@ -1,6 +0,0 @@
-public class prova {
-
-public static void main(String[] args) {
-        System.out.println("Hello World!");
-    }
-}
Index: src/Porfiri/Bomba.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Porfiri;\r\n\r\nimport Taruffi.Disegnabile;\r\nimport java.awt.*;\r\n\r\npublic class Bomba implements Disegnabile{\r\n\r\n    int x,y; //posizione della bomba\r\n    int raggio; //raggio della bomba\r\n    int timer; //timer della bomba\r\n    Image[] Sprite; //immagini dell'animazione\r\n    int indiceAnimazione; //indice dell'immagine corrente\r\n\r\n    public Bomba(int x, int y, int raggio, int timer, Image[] Sprite) {\r\n        this.x = x;\r\n        this.y = y;\r\n        this.raggio = raggio;\r\n        this.timer = timer;\r\n        this.Sprite = Sprite;\r\n        this.indiceAnimazione = 0;\r\n    }\r\n\r\n    public void Esplodi(){\r\n        //TODO: implementare l'esplodere della bomba\r\n    }\r\n\r\n    public Image nextSprite(){\r\n        return Sprite[indiceAnimazione++ % Sprite.length];\r\n    }\r\n\r\n\r\n    @Override\r\n    public void disegna() {\r\n        //TODO implementare il disegno della bomba\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Porfiri/Bomba.java b/src/Porfiri/Bomba.java
--- a/src/Porfiri/Bomba.java	
+++ b/src/Porfiri/Bomba.java	
@@ -11,10 +11,9 @@
     Image[] Sprite; //immagini dell'animazione
     int indiceAnimazione; //indice dell'immagine corrente
 
-    public Bomba(int x, int y, int raggio, int timer, Image[] Sprite) {
+    public Bomba(int x, int y, int timer, Image[] Sprite) {
         this.x = x;
         this.y = y;
-        this.raggio = raggio;
         this.timer = timer;
         this.Sprite = Sprite;
         this.indiceAnimazione = 0;
